### PR TITLE
Make SensioLabs Insight happy

### PR DESCRIPTION
**Pull request by meyerbaptiste**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -80,4 +80,4 @@ When you open a Pull Request to the PostmanGeneratorBundle project, you agree to
 Be sure to you have the right to do that (if you are a professional, ask your company)!
 
 If you include code from another project, please mention it in the Pull Request description and credit the original
-author.
\ No newline at end of file
+author.
```
Comment (vincentchalamon) [inline CONTRIBUTING.md]: What is the change? Can't see it…

Comment (meyerbaptiste) [inline CONTRIBUTING.md]: > No newline at end of file
😉

**Comment by meyerbaptiste** [comment]
Tests are now green @vincentchalamon!

**Comment by vincentchalamon** [comment]
Once PR merged, please release v1.0.4

**Pull request merged**